packages/lit-html/src/test/directives/unsafe-html_test.ts

```diff
@@ -27,6 +27,38 @@ suite('unsafeHTML directive', () => {
     );
   });
 
+  test('renders nothing', () => {
+    render(html`<div>before${unsafeHTML(nothing)}after</div>`, container);
+    assert.equal(
+      stripExpressionMarkers(container.innerHTML),
+      '<div>beforeafter</div>'
+    );
+  });
+
+  test('renders noChange', () => {
+    render(html`<div>before${unsafeHTML(noChange)}after</div>`, container);
```

Comment:
It looks like in 1.x this `unsafeHTML(noChange)` rendered `[object Object]`. I suggest we change this, but it should be doc'd as a breaking change.

Reply:
Added in https://github.com/lit/lit.dev/pull/454, PTAL